package.json

```diff
@@ -35,5 +35,8 @@
   },
   "optionalDependencies": {
     "typescript": "^5.7.2"
+  },
+  "dependencies": {
+    "json5": "^2.2.3"
```

Comment:
need to be at workspace/pakcge level `npm add -w="packages/json5"`

Reply:
thanks I this area added in packages